doc/task-hierarchy.md

```diff
@@ -112,8 +110,6 @@
 - `Sentiment Analysis`
 - `Sorting`
 - `Stance Detection`
-- `Story Completion`
```

Comment:
`Story Completion` is not a redundant category. Some tasks are to choose a part of a story from the options. So, we can't use text generation there. Maybe we can drop `Text Generation -> Sentence Generation -> Story Completion` and use both `Text Generation` and `Story Completion` in the generation tasks.

Reply:
ok let's keep both